packages/docusaurus-theme-classic/src/theme/BlogPostPage/index.tsx

```diff
@@ -65,21 +35,15 @@ function BlogPostPageContent(props: Props): JSX.Element {
     <BlogLayout
       sidebar={sidebar}
       toc={
-        !hideTableOfContents && BlogPostContents.toc.length > 0 ? (
+        !hideTableOfContents && toc.length > 0 ? (
           <TOC
-            toc={BlogPostContents.toc}
+            toc={toc}
             minHeadingLevel={tocMinHeadingLevel}
             maxHeadingLevel={tocMaxHeadingLevel}
           />
         ) : undefined
       }>
-      <BlogPostItem
-        frontMatter={frontMatter}
-        assets={assets}
-        metadata={metadata}
-        isBlogPostPage>
-        <BlogPostContents />
-      </BlogPostItem>
+      <BlogPostItem>{children}</BlogPostItem>
```

Comment:
wonder if `<BlogPostItem>` should be applied here or in parent 🤷‍♂️ 

The BlogLayout + toc logic is a bit duplicated between blog/changelog currently